components/profile/OccupationSelect.tsx

```diff
@@ -45,3 +46,15 @@ export const OccupationSelect = ({
     </div>
   );
 };
+
+const encodeSelection = (occupation: Set<string>, id: string) => {
```

Comment:
Chvilku jsem tady bádal nad tím, jestli nemít samostatně něco jako `flip<T>(values: Set<T>, value: T)` a samostatně `encode(values: Set<string>)`, ale je to samozřejmě naprostá podružnost :)

Reply:
Jakože 2 obecný funkce:
1. `flip` by buď odebrala z množiny nebo přidala do množiny prvek.
2. `encode` by převáděla množinu na string?

Reply:
Přesně tak. Když pak budu číst `encodeSelection(flip(occupationSet, id))`, je mně to jasnější.